packages/api/src/apps/api/app.ts

```diff
@@ -44,6 +44,7 @@ async function run(env: ProcessEnv) {
     },
     synthPrices: {
       latest: {},
+      history: {},
```

Comment:
what happens if the API is restarted? the history will be cleared out, right? how does the path to adding in a DB work with this exactly? I know you've probs explained it before in a previous PR but I'd just like to understand a bit better.

Reply:
Currently everything is in memory, so it will lose history. The history table here is set up to use what im calling a "store", which can be implemented with a google datastore ( currently using in memory store). This shouldn't change any of the logic in the api package, as the store conforms to a fixed interface.  So the idea is that we can implement a new store and gain persistence without having to change much else with business logic.